field/src/goldilocks_field.rs

```diff
@@ -118,22 +118,6 @@ impl Field for GoldilocksField {
         reduce128(n)
     }
 
-    #[inline]
```

Comment:
The diff is misleading here.  Logically, moved `from_noncanonical_u64` and `from_noncanonical_i64`, but github guesses wrong.